cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -137,3 +141,17 @@ func normalizeDotNetSecret(key string) string {
 	// ":".
 	return strings.ReplaceAll(key, "__", ":")
 }
+
+func findProjectFile(path string) ([]string, error) {
+	files, err := filepath.Glob(path + "/*proj")
```

Comment:
This implementation does match up closely with `dotnet`, which has the following [written](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build#arguments):

> If a project or solution file isn't specified, MSBuild searches the current working directory for a file that has a file extension that ends in either proj or sln and uses that file.

However, my crude approach may have misled you, and I think in practice this ends up matching files without file extensions, like "myfunnyproj" which is technically not "my.funnyproj".

Reply:
Can you share a test example specifically against `myfunnyproj`? This is a file with no file extensions.